tests/success/SuccessTest.bat

```diff
@@ -0,0 +1,49 @@
+@echo off
+REM ---------------------------------------------------
+REM testThatGreeterReturnsTheCorrectGreeting
+REM ---------------------------------------------------
+
+REM Initalize result variable
+set "slug=Success"
+set "result="
+set "expected=Hello, World!"
+
+REM Run the program and store the output in the result variable
+echo.Looking for "%~dp0%slug%.bat" > log.txt
+if exist %~dp0%slug%.bat (
+    REM If the file exists in the full path, run it from there
+    echo.Found "%~dp0%slug%.bat" > log.txt
+    CALL %~dp0%slug%.bat > stdout.bin
+    set /p result=<stdout.bin
+    del stdout.bin
+) else (
+
+    echo.Could not find "%~dp0%slug%.bat" > log.txt
+    echo.Looking for "%slug%.bat" > log.txt
+
+    if exist %slug%.bat (
+        REM If the file exists in the wildcard search path, run it from there
+        echo.Found "%slug%.bat" > log.txt
+        CALL %slug%.bat > stdout.bin
+        set /p result=<stdout.bin
+        del stdout.bin
+    ) else (
+        REM Errorlevel = 2: The system cannot find the file specified. 
+        REM                 Indicates that the file cannot be found in specified location.
+        echo Could not find "%~dp0%slug%.bat" or "%slug%.bat"
+        exit /b 2
+    )
+)
```

Comment:
I don't fully understand why we need to relatively complex behavior. When the student downloads the exercise it is guaranteed that the test and solution files are downloaded, so you can just assume that the solution file is there next to the test file. Having the test file be simpler is also really useful for the student.

Reply:
In fact, but a second possibility was to find the way, but it may not be so necessary, now I have written a code that more students can understand, it is completely open to criticism.

Reply:
I would replace the code with:

```
CALL %slug%.bat > stdout.bin
set /p result=<stdout.bin
del stdout.bin
```

Reply:
@GroophyLifefor Be aware that no student will ever seen any of this. It runs on our servers - it's not something a student would ever download etc :)